experimental/dispatch_profiler/manifest.py

```diff
@@ -16,19 +21,21 @@ def __init__(self, operation_path, dispatch_collection):
     self.operation = dispatch_collection.operation
     self.operation_kind = self.operation.operation_kind
     self.configuration_list = dispatch_collection.configuration_list
-    self.operation_filepath = os.path.join(self.operation_path, \
+    self.operation_filepath = os.path.join(self.operation_path,
                                            self.operation.name() + ".mlir")
 
     mlir_configuration_emitter = {
         OperationKind.Matmul: EmitMatmulCompilationInfo,
-        #OperationKind.Conv2d : EmitConv2dCompilationInfo, TODO: Add conv2d
```

Comment:
Todo for Conv2D removed but not implemented here?

Reply:
Yeah, removed. We will add it back when we implement conv2d and other operations. The idea when put this comment it show what all parts will change when adding another op-kind. Now we have batch_matmul as another op_kind. We have enough different op_kind examples to follow for someone who wants to add another op_kind to the tool.